webextensions/manifest/permissions.json

```
@@ -413,6 +413,26 @@
             }
           }
         },
+        "devtools": {
+          "__compat": {
+            "description": "<code>devtools</code>",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "54"
+              },
+              "firefox_android": "mirror",
```

```suggestion
              "firefox_android": {
                "version_added": false
              },
```

The `devtools_page` feature and the associated `devtools` permission are desktop-only.